tests/unit/test_bucket.py

```diff
@@ -37,4 +37,4 @@ def test_upload_directory(self, images_dir, boto_session):
 
     def test_fail_upload_directory(self, images_dir):
         with pytest.raises(ClientError):
-            assert upload_directory(images_dir, self.test_bucket_name, s3_path)
+            assert upload_directory(images_dir, self.test_bucket_name, s3_path)
```

Comment:
Is this change intentional?

Reply:
This should have been included in the main branch. I am not sure what changed vs the original code.